literature.bib

```diff
@@ -10522,6 +10866,35 @@ @article{BSRC10
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2009 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{FHS:SPLC09,
+	title = {{Inferring Information From Feature Diagrams to Product Line Economic Models}},
+	author = {David Fern{\'{a}}ndez{-}Amor{\'{o}}s and Ruben Heradio Gil and
+               Jos{\'{e}} Antonio Cerrada Somolinos},
```

Comment:
We usually have everything in one line. I am fine with both. Should we introduce a rule for that?